apps/meteor/tests/e2e/channel-management.spec.ts

```diff
@@ -57,69 +57,81 @@ test.describe.serial('channel-management', () => {
 		await expect(page.getByRole('button', { name: 'Start call' })).toBeFocused();
 	});
 
-	// FIXME: bad assertion
-	test('should add "user1" to "targetChannel"', async () => {
+	test('should add user1 to targetChannel', async () => {
 		await poHomeChannel.sidenav.openChat(targetChannel);
 		await poHomeChannel.tabs.btnTabMembers.click();
 		await poHomeChannel.tabs.members.showAllUsers();
 		await poHomeChannel.tabs.members.addUser('user1');
 
-		await expect(poHomeChannel.toastSuccess).toBeVisible();
+		await expect(poHomeChannel.getSystemMessageByText('added user1')).toBeVisible();
```

Comment:
Maybe we could add an extra validation to ensure the user is in the room, WDYT? 
e.g: expect that the user is in the channel list